packages/typescript/README.md

```diff
@@ -8,7 +8,7 @@ The `@jamsocket/javascript` library is composed of
 - `/react` uses the `client` library to give you the same functionality in simple React hooks
 - `/socketio` lets you connect to a socketio server in your session backend with React hooks
 
-[View the open source library on Github.](https://github.com/drifting-in-space/jamsocket-javascript)
+[View the open source library on Github.](https://github.com/jamsocket/jamsocket)
```

Comment:
should this be further linked to packages/typescript

Reply:
Good call